### ISSUE TITLE
Staging Review finding: Images are not conveyed to assistive technology

### ISSUE THREAD

**Issue by jasonday** [department-of-veterans-affairs/va.gov-team#122539]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Reapers Team
**Product:** VRE Ch31 Eligibility & Entitlement
**Feature:** VRE Ch31 Eligibility & Entitlement

## Findings details
**VA.gov Experience Standard - issue:** [User can't perceive an element.](https://design.va.gov/about/experience-standards/#user-can-perceive-all-meaningful-elements)
**VA.gov Experience Standard - category:** [Comprehension](https://design.va.gov/about/experience-standards/#category-comprehension)
**Launch-blocking:** Yes
**Design System review:** No
**Collab Cycle Reviewer:** @jasonday (Accessibility)

### Description

Checkmarks as part of the Basic Eligibility Criteria are not conveyed to assistive technology.

<details><summary>Link, screenshot or steps to recreate</summary>

<img width="426" height="206" alt="checkmarks" src="https://github.com/user-attachments/assets/805bb721-cd07-4e9f-8bc4-b0ed387edc7a" />

</details>

## Recommended action

These checkmarks provide context to sighted users, and must be exposed to assistive technology, so that AT users have an equivalent experience and can understand the associated context. Provide a text alternative such as "eligible".

## References

- **Accessibility Defect Severity:** [1: Critical. Must be fixed before launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-1)
- **WCAG Success Criteria:** [WCAG 1.1.1 Non-text Content - Level A](https://www.w3.org/TR/WCAG22/#non-text-content)
- **Modality:** Screen reader
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://dj540s05.optimalworkshop.com/questions/f54355271969f160ef41182ba9ceb61c) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue; or if you believe the finding should not be fixed; comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by KatyaShick** [comment]
https://github.com/department-of-veterans-affairs/va.gov-team/issues/121247  - there is a component of this finding that was not a problem from our team. We created a support ticket which can be found here.
